CHANGELOG.md

```diff
@@ -12,16 +12,20 @@ Initial release of Arcadia-Science/seqqc, created with the [nf-core](https://nf-
 - Local module to download sourmash GTDB database and human signature
 - Workflow to run sourmash gather to detect routine contamination
 - Workflow to run sourmash compare to determine sequence similarity between samples
-- Integration of sourmash outputs into the MultiQC outputs
+- Integration of sourmash outputs into the MultiQC outputs (when run with docker profile only)
+- Conda, Docker, and Singularity integration
 - Documentation for the pipeline
+- MultiQC interpretation documentation
+- Test profiles for different sequencing chemistries
+- Updates to email template
 
 ### `Fixed`
 
 ### `Dependencies`
 
 - `fastqc=0.11.9`
-- `multiqc=1.13` # currently installed from a dev branch, https://github.com/taylorreiter/MultiQC/tree/ter/add-sourmash-gather commit 47808ae
-- `sourmash=4.5.0`
-- `wget=1.20.1`
+- `multiqc=1.14` # currently installed from a dev branch, https://github.com/taylorreiter/MultiQC/tree/ter/add-sourmash-gather commit 47808ae
```

Comment:
you can just link to the commit directly maybe?

Reply:
good idea! will do
```suggestion
- `multiqc=1.14` # currently installed from a [dev branch](https://github.com/taylorreiter/MultiQC/tree/47808aea9b05305e82927ba51f8e266ca0b919c6)
```